code/datums/achievements/antagmastery.dm

```diff
@@ -0,0 +1,112 @@
+/datum/award/achievement/antagmastery
+	category = "Antag Mastery"
+	reward = 400
+
+/datum/award/achievement/antagmastery/on_unlock(mob/user)
+	. = ..()
+	for(var/datum/award/achievement/antagmastery/A in SSachievements.antag_mastery_list)
+		if(!user.client?.player_details.achievements.get_achievement_status(A))
+			return
+	user.client?.give_award(/datum/award/achievement/antagmastery/super, user)
```

Comment:
Will this not always give them the antag master achievement since there is no things of type /datum/award/achievement/antagmastery in SSachievements.antag_mastery_list?